examples/dao/assets/dao-fund-lsig.py

```diff
@@ -0,0 +1,54 @@
+import sys
+sys.path.insert(0,'..')
+
+from algobpy.parse import parse_params
+from pyteal import *
+
+def proposal_lsig(ARG_DAO_APP_ID):
```

Comment:
can we rename this function ?

Reply:
maybe `treasury_lsig`

Reply:
or `dao_fund_lsig` ;)

Reply:
Missed this one. Renamed to `dao_fund_lsig` (it's the file name as well)